common-lib/src/cmr/common/util.clj

```diff
@@ -120,6 +120,22 @@
   [v]
   (when v (str/upper-case v)))
 
+(defn capitalize-words
+  "Capitalize every word in a string"
+  [s]
+  (->> (str/split (str s) #"\b")
+       (map str/capitalize)
+       str/join))
+
+(defn match-enum-case
+  "Given a string and a collection of valid enum values, return the proper-cased
+   value from the enum."
```

Comment:
I like this function. Please update the doc string to indicate that it returns the case insensitive mapping of the value in the enum list.

Reply:
This function takes a value, and if it's valid, returns the corresponding one in the given enum to guarantee that its casing is correct. I believe the name is correct, but the docstring could be expanded on. Done